site/source/tutorials/accessibility.md

```diff
@@ -0,0 +1,233 @@
+___
+
+### Calendar
+
+The Calendar widget renders a date grid with dropdown month and year pickers. Both the month and year dropdowns use a popup button, and control focus between the button and popup when opened and closed. Focusable items in hidden popups are removed from the focus order.
+
+The date grid itself uses a `<table>` element with weekdays as the column headers. Hidden accessible text is provided for the previous/next pagination arrows that only visibly display an icon. The date table is labelled by a hidden `<label>` element showing the current month and year. It has `aria-live="polite"` so screen readers will be notified when the displayed month changes, e.g. when a user navigates to the next month via arrow keys.
+
+##### Keyboard Interaction
+Items in the focus order for this widget in its initial state are the buttons for the month and year pickers, the currently focused date, and the previous and next month buttons. The following keyboard interactions are available when focus is within the date grid:
+- **Left arrow**: Moves focused date to the previous date
+- **Right arrow**: Moves focused date to the next date
+- **Down arrow**: Moves focused date to the same day of the next week
+- **Up arrow**: Moves focused date to the same day of the previous week
+- **Enter & Space**: Selects the focused date
+- **Page Up**: Moves focused date to the first day of the month
+- **Page Down**: Moves focused date to the last day of the month
+
+When the focused date moves to a day in the next or previous month, the date grid will refresh.
+
+##### A11y properties
+- `labels`: The Calendar widget has a number of internal descriptive labels, and this property allows their text to be customized
+
+___
+
+### Dialog
+
+The Dialog widget creates a controlled dialog element. Controlled focus is not yet implemented, but will be added along with the focus manager.
+
+##### A11y properties
+- `closeText`: Provides hidden, accessible text for the icon close button. Defaults to "close dialog."
+- `role`: The default role for the dialog container is `dialog`, but it can be changed to `alertdialog` through this property.
+
+___
+
+### SlidePane
+
+SlidePane is similar to dialog in that it creates a modal that overlays page content. It will need similar programmatic focus control, pending completion of the focus manager.
+
+##### A11y properties
+- `closeText`: Provides hidden, accessible text for the icon close button. Defaults to "close pane."
+
+___
+
+### TabController
+
+TabController creates a set of tabs that control the content of a container. The tabs use `role="tab"` and are associated with the content through `aria-controls` and `aria-labelledby`. Only one tab is in the tab order, and arrow keys are used to switch between tabs.
```

Comment:
I don't know what " Only one tab is in the tab order," is telling me.

Reply:
I changed all references to the widget tabs into "tab buttons" to try to avoid confusion with tab order/tab stops.